bulk/detail/cuda_launcher/triple_chevron_launcher.hpp

```diff
@@ -177,9 +181,13 @@ class triple_chevron_launcher<block_size_,Function,false> : protected triple_che
 
 #if __BULK_HAS_CUDART__
 #  ifndef __CUDA_ARCH__
-          cudaConfigureCall(dim3(num_blocks), dim3(block_size), num_dynamic_smem_bytes, stream);
-          cudaSetupArgument(static_cast<const task_type*>(parm.get()), 0);
-          bulk::detail::throw_on_error(cudaLaunch(super_t::global_function_pointer()), "after cudaLaunch in triple_chevron_launcher::launch()");
+          // create an array of void*s which point to each argument
+          void* pointers_to_args[] = {reinterpret_cast<void*>(const_cast<task_type*>(parm.get()))};
+
+          bulk::detail::throw_on_error(
+            cudaLaunchKernel(super_t::global_function_pointer(), dim3(num_blocks), dim3(block_size), pointers_to_args, num_dynamic_smem_bytes, stream),
```

Comment:
To avoid the same error reported in the previous comment, we just need to add the ``reinterpret_cast<void*>`` here too:
``` 
bulk::detail::throw_on_error(
            cudaLaunchKernel(reinterpret_cast<void*>(super_t::global_function_pointer()), dim3(num_blocks), dim3(block_size), pointers_to_args, num_dynamic_smem_bytes, stream),
            "after cudaLaunchKernel in triple_chevron_launcher::launch()"
          );
```